src/pages/users/cli/governance/index.mdx

```diff
@@ -1,10 +1,62 @@
 ---
 title: Governance
-pageType: sub-category
 ---
 
-import { CurrentPageNavigationSections } from "~/components/shared";
+import { GovParams } from "~/components/Docs";
 
-<div className="flex flex-col gap-20 sm:gap-[120px]">
-  <CurrentPageNavigationSections />
-</div>
+import { Alert } from "~/components/shared";
+
+## Overview
+
+ZetaChain uses the standard Cosmos SDK [`gov` governance
+module](https://docs.cosmos.network/v0.46/modules/gov/) to manage the network.
```

Comment:
Should we use https://docs.cosmos.network/v0.50/build/modules/gov now that we bumped the cosmos version?

Reply:
@CharlieMc0

Reply:
Yes. Can you add a new issue to review docs and update everything to v50? Could be a good tech ambassador issue.

Reply:
Created issue: https://github.com/zeta-chain/docs/issues/546